components/prism-llvm.js

```diff
@@ -0,0 +1,12 @@
+(function(Prism) {
+	Prism.languages.llvm = {
+		'comment': /;.*/,
+		'string': /"[^"]*"/,
```

Comment:
Should the `c` prefix be also highlighted? 
E.g. `c"foo"`

And please make it greedy or strings like `"oh ; no"` will not be highlighted correctly.